server/src/resolvers/Mutation/pyrog.ts

```diff
@@ -111,7 +111,14 @@ export const pyrogMutation = {
 
   updateResource: checkAuth(forwardTo('binding')),
 
-  updateAttribute: checkAuth(forwardTo('binding')),
+  async updateAttribute(parent, { id, data }, context: Context) {
```

Comment:
nit: this function does not need to be async (and you don't need to return `await context.client....`, you can just drop the await I think)

Reply:
You're right! In that case, I should also remove the `await`s for the other functions too, am I right?

Reply:
yes you can, it's always good to clean the project when you notice something isn't right at one place.

Reply:
👍